pkg/common/args.go

```diff
@@ -48,6 +48,12 @@ func PrintInfo(clientSet *kubernetes.Clientset, config *rest.Config) {
 // ValidateArgs validates the arguments passed to the program
 // and creates the output directory if it doesn't exist
 
+func SetDefaultNamespace() {
```

Comment:
@Bharadwajshivam28 if possible can you  add unit test for this?

Reply:
> @Bharadwajshivam28 if possible can you add unit test for this?

I have implemented unit test but can you please elaborate this?
I will be happy to make the changes @reetasingh

Reply:
you can add the unit test for this in args_test.go